src/main/java/com/naveenautomation/Pages/ChangePasswordPage.java

```diff
@@ -8,7 +8,7 @@
 public class ChangePasswordPage extends Page {
 
 	
-	private static final String PAGE_URL="/password";
+	private static final String PAGE_URL="account/password";
```

Comment:
if account is common in all URL, it should be in base URL and not in page url

Reply:
URL changed as recommended.